### PR TITLE
feat(multiples): add size variable

### PR DESCRIPTION
**Pull request by iacobson**
In order to be able to calculate estimated returns for multiples such as doubles when we have 3 selections, needed to add a size variable to the `multiple` function.

The size defaults to the selections length.

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -44,8 +44,8 @@ function accumulator({
   return calculateAccumulator(selections, stake, ew, idx, place);
 }
 
-function calculateMultiple(selections, stake, ew = false, fullCover = false) {
-  const count = selections.length;
+function calculateMultiple(selections, stake, ew = false, fullCover = false, size = null) {
+  const count = size || selections.length;
```
Comment (oherst) [inline lib/main.js]: Can you define `selections.length` as the default param for size?

Edit: In fact, if you define `count = selections.length` in the args, you don't need the `count` declaration inside the function at all

Comment (iacobson) [reply lib/main.js]: 👍

File: lib/main.js [hunk 2]
```diff
@@ -44,8 +44,13 @@ function accumulator({
...
 }
 
-function calculateMultiple(selections, stake, ew = false, fullCover = false) {
-  const count = selections.length;
+function calculateMultiple(
+  selections,
+  stake,
+  ew = false,
+  fullCover = false,
+  count = selections.length
```
Comment (hopeshigh) [inline lib/main.js]: How come this has moved from inside the function to outside of it?

Comment (iacobson) [reply lib/main.js]: there was a suggestion from Ollie.
so we are passing a new variable size

quoting from Ollie:

change this:
```
function calculateMultiple(selections, stake, ew = false, fullCover = false, size = null) {
      const count = size || selections.length;
```
to:
```
function calculateMultiple(selections, stake, ew = false, fullCover = false, count = selections.length) {
```

I thought it's a good idea. What do you think?

Comment (hopeshigh) [reply lib/main.js]: Ah okay, I get that.

My concern isn't so much with the implementation, it's more the arity creep. So we went from `calculateMultiple/4` to `calculateMultiple/5` and admittedly some are default variables.

Would we know the values when they appear in the `multiple` function? 

If so we could use this to work out whether it needs default values or not then we can pass in one argument as an object to this function.

**Review by oherst — APPROVED** [review]
Looks good.

**Pull request merged**